tools/autograph-monitor/monitor.go

```diff
@@ -90,9 +87,9 @@ func main() {
 
 	if os.Getenv("LAMBDA_TASK_ROOT") != "" {
 		// we are inside a lambda environment so run as lambda
-		lambda.Start(Handler)
+		lambda.Start(func() error { return Handler(conf, http.DefaultClient) })
```

Comment:
A little nit-picky. I think `Handler`'s only purpose was to be a lambda-compatible wrapper around `monitor`. But now we're wrapping the wrapper with this function. Why not just get rid of `Handler` now?

Reply:
It has that GC call inside of it and that might actually still matter, so I didn't want to adjust or remove it.

Reply:
And moving all that code out here just to capture it all again feels pretty not great.

Reply:
(Cuz the point of this wrapper func is to capture the local variable `conf` in `main`.)

Reply:
Hm, are you maybe asking for `conf` to still be a global that Handler knows to look up? I was hoping to avoid needing mutable globals after this change.

Reply:
I guess this is a moot point as the lambda is going away soon anyway.